rholang/examples/tut-philosophers.rho

```diff
@@ -0,0 +1,14 @@
+new north, south, knife, spoon in {
+    north!(knife) |
+    south!(spoon) |
+    for (knf <- north; spn <- south) {
+        "Philosopher 1 Utensils: ".display(knf, ", ", spn, "\n") |
```

Comment:
this looks like strings have a display method; i.e. that every string carries the capability to print itself. Am I reading that correctly?